app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectFragment.kt

```diff
@@ -12,6 +12,7 @@ import android.net.wifi.SupplicantState
 import android.net.wifi.WifiManager
 import android.os.Build
 import android.os.Bundle
+import android.os.Message
```

Comment:
Where is this being used? Is this extra import?

Reply:
No this is not